src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1322,6 +1322,9 @@ def __init__(self,
                  enable_prometheus_exporter: Optional[bool] = True,
                  bdevs_per_cluster: Optional[int] = 32,
                  verify_nqns: Optional[bool] = True,
+                 allowed_consecutive_spdk_ping_failures: Optional[int] = 1,
```

Comment:
I guess it's up to you, but allowing only 1 consecutive failure by default feels a bit odd to me. Doesn't that imply it will just bail out as soon as one of the pings fails It seems like if you use the defaults this would cause no change in the behavior.

Reply:
@adk3798 I kept the current implementation which will abort after one ping failure. But I allow more flexibility by setting this field to a bigger number.